src/org/daisy/dotify/formatter/impl/page/PageSequenceBuilder2.java

```diff
@@ -592,7 +585,15 @@ private PageImpl nextPageInner(
                     cbl = gr.getLineProperties().getBlockLineLocation();
                 }
                 // Add the body rows to the page.
-                bc = addRows(head, current, bc);
+                if (firstPageContentRow > 0) {
+                    bc = addRows(head.subList(0, firstPageContentRow), current, bc, false);
+                    if (!current.getDetails().getMarkers().isEmpty()) {
+                        throw new RuntimeException();
```

Comment:
Hi @bertfrees 

I usually dislike throwing if there is a way to handle the issue with some output. But in this case, it seems we forgot to give a description of why this is a case where we can't render the content and we must fail the rendering. 

```
throw new RuntimeException("We can't render this content because ... ");
```

Best regards
Daniel

Reply:
You're right. I did this in a haste. The exception is not really needed I guess. Before we didn't complaint either. I just thought having markers in a `any-resumed` (and `any-interrupted` etc.) didn't make sense so we should avoid it. (The same with `id`s by the way, which I'm not testing here.) The proper way to enforce this is in the OBFL parser I think (and even better would be to do it on the OBFL schema level).

I'm not completely sure that a `marker` in a `any-resumed` would _never ever_ make sense though. It sure doesn't make sense for page numbers, but there can also be other kinds of markers that we didn't think of. Shall we add this restriction to the OBFL parser until a use case is found for which we need to be more relaxed? This is an approach we've taken before too.

Reply:
I've added a new commit that does what I described above.